src/leap/mx/mail_receiver.py

```diff
@@ -145,7 +146,7 @@ def _start_watching_dir(self, dirname, recursive):
         directory = filepath.FilePath(dirname)
         try:
             if not directory.isdir():
-                raise OSError("Not a directory: '%s'" % directory.path)
+                mkdir_p(dirname)
```

Comment:
I have a pull request to change leap-mx from running as root: https://github.com/leapcode/leap_mx/pull/78

The leap platform will create the correct user/group before the package is installed.